server/infra/database/PlanterRepository.ts

```diff
@@ -1,13 +1,69 @@
 import FilterOptions from 'interfaces/FilterOptions';
 import Planter from 'interfaces/Planter';
+import HttpError from 'utils/HttpError';
 import BaseRepository from './BaseRepository';
 import Session from './Session';
 
+enum RegionTypes {
+  country = 6,
+  continent = 5,
+}
 export default class PlanterRepository extends BaseRepository<Planter> {
   constructor(session: Session) {
     super('planter', session);
   }
 
+  async getById(id: string | number) {
```

Comment:
Can you combine the `sql` for this method into one string so that you don't have to do queries twice for a single API request? I also tested your sql string for `planter__d = 1` and I am getting the region object containing `Africa and Brazil`. As far as I can tell it should return `North America and United States`.  Even if this specific planter planted in two different continents the country should at least be consistent with the continent. I am afraid you are sampling two separate trees for country and continent.

P.S. It's usually a good idea not to include multiple issues in one pull request. If you feel inclined to you can leave this issue to me, I have been working on the issue 107 already so I have an idea on how to get country and continent from a single tree planted by a planter.